rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,424 @@
+                                                    'fontSize' : '12',
+                                                    'fontColor':'gray',
+                                                    'showBackground':'false',
+                                                    'position' : {'x' : xyz[i][0],
+                                                                  'y' : xyz[i][1],
+                                                                  'z' : xyz[i][2]
+                                                               }
+                                                   })
+        except:
+            pass
+    
+    try:
+        bg_color_selected = globals()['background_'+uid].value
+        globals()['view_'+uid].setBackgroundColor(bg_color_selected)
+    except:
+        pass
+    
+    # To do : Add protein in viewer
+    if 'protein_'+uid in globals():
+        pdb = Chem.MolToPDBBlock(globals()['protein_'+uid], flavor=0x20|0x10)
+        globals()['view_'+uid].addModel(pdb,'pdb')
+        globals()['view_'+uid].setStyle({},{'cartoon':{'color': 'spectrum'}})
+    
+    
+    # zoomTo does not work well for surface and label... so, zoomTo should not be default settings
+    #view.zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+def handle_change(change):
+    """This function handles all the interactive widgets except buttons and 3Dmol.js viewer"""
+    update3D(change.owner._model_id)
+    
+def handle_start_button(b):
+    """This function handles start button."""
+    b.icon='check'
+    b.description="Done!"
+    update3D(b._model_id)
+    
+def handle_zoomTo_button(b):
+    """This function handles zoomTo button"""
+    uid=globals()['rdkit_wg_dict'][b._model_id]
+    globals()['view_'+uid].zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+    
+def ShowConformers3D(Mol, protein = None,
+                     useDrawAs = False, drawAs=None, 
+                     propPanel = False, 
+                     colorPanel = False, 
+                     labelPanel = False):
+    
+    """This function initiates required widgets and 3Dmol.js viewer"""
+    
+    # Common suffix for widgets
+    uid=str(time.time()).replace('.','')
+    
+    # Required global objects
+    globals()['rdkit_mol_select_'+uid] = list()
+    globals()['rdkit_conf_select_'+uid] = list()
+    
+    if 'rdkit_wg_dict' not in globals():
+        globals()['rdkit_wg_dict'] = dict()
+    
+    
+    # Right hand panel (widgets)
+    wgListBox=list()
+    itemLayout=Layout(display='flex', flex_flow='row', justify_content='space-between')
+    
+    
+    # To do : Add protein
+    if protein is not None:
+        #print 'adding protein'
+        globals()['protein_'+uid] = protein
+    
+    
+    # molecules and conformers
+    globals()['moldict_'+uid] = ProcessMolContainingObj(Mol)
+    keys=list(globals()['moldict_'+uid].keys())
```

Comment:
`globals()['moldict_'+uid] = ProcessMolContainingObj(Mol)` here I am unwilling to keep the `ProcessMolContainingObj` function. So, `OrderedDict` has NOT been implemented.